integrations/extensions/docs/elasticsearch-install-and-setup/federated_search.md

```diff
@@ -0,0 +1,108 @@
+# Federated Search in Elasticsearch
+
+This guide dives into performing a single search query across multiple indexes within an Elasticsearch cluster. It assumes these indexes share the same relevant fields for your search results.
+
+## Methods for Multi-Index Search
+
+There are several approaches to searching across multiple indexes in Elasticsearch:
+
+### 1. Comma-Separated Indices:
```

Comment:
Thanks, Reza! I think another option is to use an alias where multiple indices share the same alias.

Reply:
Thanks, Zach! That's right. I can add that too. The only reason I didn't was to keep it simple as that requires an extra step of creating the alias as well.